MariaOlivenbaum/HW9/9.1.py

```diff
@@ -0,0 +1,64 @@
+from random import randint
+from tkinter import *
+from tkinter import ttk
+from tkinter import messagebox
+
+root = Tk()
+root.geometry('600x400')
+root.title("Number guesser")
+
+
+def read_user_number():
+    user_number = user_field.get()
```

Comment:
qq. How to transfer functions to other modules from main, and at the same time be able to access fields (Enter()) and buttons (e.g.: user_field.get())